src/components/checkbox/Checkbox.js

```diff
@@ -46,12 +46,17 @@ export class CheckBoxComponent extends BaseComponent {
     this.label = this.ce('label', 'label', {
       class: 'control-label'
     });
+
+    // Create the SPAN around the textNode for better style hooks
+    this.labelText = this.ce('span', 'span');
```

Comment:
Can you change the first string here to be descriptive id of what this is. I would call it ```labelSpan``` so change to this please.

```
this.labelText = this.ce('labelSpan', 'span');
```

This label is used to hook into the rendering from an external library so that it can control every element that is created within the renderer. Having a descriptive label helps that external library know what it is changing.